packages/cli/docs-markdown-utils/src/parseImagePaths.ts

```diff
@@ -227,18 +227,31 @@ export function replaceImagePathsAndUrls(
     let offset = 0;
 
     function mapImage(image: string | undefined) {
-        if (image != null && !isExternalUrl(image) && !isDataUrl(image)) {
-            try {
-                const fileId = fileIdsMap.get(AbsoluteFilePath.of(image));
-                if (fileId != null) {
-                    return `file:${fileId}`;
-                }
-            } catch (e) {
-                // do nothing
-                return;
-            }
+        if (image == null || isExternalUrl(image) || isDataUrl(image)) {
+            return undefined;
         }
-        return;
+
+        // Check if it's an absolute path
+        if (image.startsWith("/")) {
```

Comment:
this won't work on windows, we'll want to make sure that our logic is compatible across different operating systems

Reply:
https://github.com/fern-api/fern/blob/05fd0adfcbe44bb3ef2ff1637eeb331821b6d06a/.github/workflows/ci.yml#L143-L173 you could write a simple test that make ssure `fern docs dev` runs on windows with a sample docs fixture